frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx

```diff
@@ -23,25 +25,20 @@ export const ParcelAttributes: React.FC<IParcelAttributes> = ({ parcelInfo, canV
   let formatAssessed;
   if (parcelInfo?.assessedLand) {
     formatAssessed = formatMoney(parcelInfo?.assessedLand);
-  } else if (parcelInfo?.evaluations?.length >= 1) {
+  } else if (!!getCurrentYearEvaluation(parcelInfo?.evaluations, EvaluationKeys.Assessed)) {
     formatAssessed = formatMoney(
-      parcelInfo?.evaluations.sort((a, b) => compareDate(a.date, b.date)).reverse()[0].value,
+      getCurrentYearEvaluation(parcelInfo?.evaluations, EvaluationKeys.Assessed)?.value,
     );
   } else {
     formatAssessed = '';
   }
-
-  let improvements = 0;
+  let improvements = '';
   if (parcelInfo?.assessedBuilding) {
-    improvements = parcelInfo?.assessedBuilding;
-  } else if (parcelInfo?.buildings?.length >= 1) {
-    parcelInfo.buildings.forEach((building: IBuilding) => {
-      if (building.evaluations?.length >= 1) {
-        improvements += +building.evaluations
-          .sort((a: IEvaluation, b: IEvaluation) => compareDate(a.date, b.date))
-          .reverse()[0].value;
-      }
-    });
+    improvements = formatMoney(parcelInfo?.assessedBuilding);
+  } else if (!!getCurrentYearEvaluation(parcelInfo?.evaluations, EvaluationKeys.Improvements)) {
+    improvements = formatMoney(
```

Comment:
I guess this doesn't really hurt anything, but I'm pretty sure the assessedBuilding IS the improvement from the current year.